File: AMS_基础信息组件/code/server/source/com.yss.ams.base.information.support/src/main/java/com/yss/ams/base/information/support/sys/indexdi/pojo/Indexdi.java
package com.yss.ams.base.information.support.sys.indexdi.pojo;

import com.yss.framework.api.common.co.BasePojo;

/**
 * 合规指标类型字典T_S_INDEX pojo
 *
 */
public class Indexdi extends BasePojo{
	
	private static final long serialVersionUID = -6675703938820914139L;

	private String c_INDEX_CODE = "";
	
	private String c_INDEX_NAME = "";
	
	private String c_DATA_SOURCE = "";
	
	private String c_DATA_TYPE = "";
	
	private int n_STATE = 0;
	
	private int n_ORDER = 0;
	
	private String c_NAV_TYPE = "";
	
	private int n_DETAIL = 0;
	
	private String c_KEY_CODE = "";
	
	private String c_KEY_NAME = "";
	
	private String c_IS_SYS = "";
	
	private String c_TRU = "";
	
	private String c_MODE = "";
	
	private String c_RET = "";
	
	public String getC_INDEX_CODE() {
		return c_INDEX_CODE;
	}

	public void setC_INDEX_CODE(String cINDEXCODE) {
		c_INDEX_CODE = cINDEXCODE;
	}

	public String getC_INDEX_NAME() {
		return c_INDEX_NAME;
	}

	public void setC_INDEX_NAME(String cINDEXNAME) {
		c_INDEX_NAME = cINDEXNAME;
	}

	public String getC_DATA_SOURCE() {
		return c_DATA_SOURCE;
	}

	public void setC_DATA_SOURCE(String cDATASOURCE) {
		c_DATA_SOURCE = cDATASOURCE;
	}

	public String getC_DATA_TYPE() {
		return c_DATA_TYPE;
	}

	public void setC_DATA_TYPE(String cDATATYPE) {
		c_DATA_TYPE = cDATATYPE;
	}

	public int getN_STATE() {
		return n_STATE;
	}

	public void setN_STATE(int nSTATE) {
		n_STATE = nSTATE;
	}

	public int getN_ORDER() {
		return n_ORDER;
	}

	public void setN_ORDER(int nORDER) {
		n_ORDER = nORDER;
	}

	public String getc_NAV_TYPE() {
		return c_NAV_TYPE;
	}

	public void setc_NAV_TYPE(String cNAVTYPE) {
		c_NAV_TYPE = cNAVTYPE;
	}

	public int getN_DETAIL() {
		return n_DETAIL;
	}

	public void setN_DETAIL(int nDETAIL) {
		n_DETAIL = nDETAIL;
	}

	public String getC_KEY_CODE() {
		return c_KEY_CODE;
	}

	public void setC_KEY_CODE(String cKEYCODE) {
		c_KEY_CODE = cKEYCODE;
	}

	public String getC_KEY_NAME() {
		return c_KEY_NAME;
	}

	public void setC_KEY_NAME(String cKEYNAME) {
		c_KEY_NAME = cKEYNAME;
	}

	public String getC_IS_SYS() {
		return c_IS_SYS;
	}

	public void setC_IS_SYS(String cISSYS) {
		c_IS_SYS = cISSYS;
	}

	public String getC_TRU() {
		return c_TRU;
	}

	public void setC_TRU(String cTRU) {
		c_TRU = cTRU;
	}

	public String getC_MODE() {
		return c_MODE;
	}

	public void setC_MODE(String cMODE) {
		c_MODE = cMODE;
	}

	public String getC_RET() {
		return c_RET;
	}

	public void setC_RET(String cRET) {
		c_RET = cRET;
	}
}

File: AMS_基础信息组件/code/client/source/YssInformation.Support/Sys/Dictionary/Pojo/Indexdi.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using FAST.Common.Service.Pojo.Base;
using Newtonsoft.Json;

////namespace FAST.Platform.Dict.Pojo
////namespace YssInformation.Sys.Dictionary.Pojo
namespace YssInformation.Support.Sys.Dictionary.Pojo
{
    /// <summary>
    /// 合规指标类型字典 POJO
    /// </summary>
    public class Indexdi : BasePojo
    {
        /// <summary>
        /// c_INDEX_CODE
        /// </summary>
        private string c_INDEX_CODE;

        /// <summary>
        /// c_INDEX_NAME
        /// </summary>
        private string c_INDEX_NAME;

        /// <summary>
        /// c_DATA_SOURCE
        /// </summary>
        private string c_DATA_SOURCE;

        /// <summary>
        /// c_DATA_TYPE
        /// </summary>
        private string c_DATA_TYPE;

        /// <summary>
        /// n_STATE
        /// </summary>
        private string n_STATE;

        /// <summary>
        /// n_ORDER
        /// </summary>
        private string n_ORDER;

        /// <summary>
        /// n_NAV_TYPE
        /// </summary>
        private string c_NAV_TYPE;

        /// <summary>
        /// n_DETAIL
        /// </summary>
        private string n_DETAIL;

        /// <summary>
        /// c_KEY_CODE
        /// </summary>
        private string c_KEY_CODE;

        /// <summary>
        /// c_KEY_NAME
        /// </summary>
        private string c_KEY_NAME;

        /// <summary>
        /// c_IS_SYS
        /// </summary>
        private string c_IS_SYS;

        /// <summary>
        /// c_TRU
        /// </summary>
        private string c_TRU;

        /// <summary>
        /// c_MODE
        /// </summary>
        private string c_MODE;

        /// <summary>
        /// c_RET
        /// </summary>
        private string c_RET;

        /// <summary>
        /// 属性: C_INDEX_CODE 
        /// </summary>
        [JsonProperty(PropertyName = "c_INDEX_CODE")]
        public string C_INDEX_CODE
        {
            set { c_INDEX_CODE = value; }

            get { return c_INDEX_CODE; }
        }

        /// <summary>
        /// 属性: C_INDEX_NAME 
        /// </summary>
        [JsonProperty(PropertyName = "c_INDEX_NAME")]
        public string C_INDEX_NAME
        {
            set { c_INDEX_NAME = value; }

            get { return c_INDEX_NAME; }
        }

        /// <summary>
        /// 属性: C_DATA_SOURCE 
        /// </summary>
        [JsonProperty(PropertyName = "c_DATA_SOURCE")]
        public string C_DATA_SOURCE
        {
            set { c_DATA_SOURCE = value; }

            get { return c_DATA_SOURCE; }
        }

        /// <summary>
        /// 属性: C_DATA_TYPE 
        /// </summary>
        [JsonProperty(PropertyName = "c_DATA_TYPE")]
        public string C_DATA_TYPE
        {
            set { c_DATA_TYPE = value; }

            get { return c_DATA_TYPE; }
        }

        /// <summary>
        /// 属性: N_STATE 
        /// </summary>
        [JsonProperty(PropertyName = "n_STATE")]
        public string N_STATE
        {
            set { n_STATE = value; }

            get { return n_STATE; }
        }

        /// <summary>
        /// 属性: N_ORDER 
        /// </summary>
        [JsonProperty(PropertyName = "n_ORDER")]
        public string N_ORDER
        {
            set { n_ORDER = value; }

            get { return n_ORDER; }
        }

        /// <summary>
        /// 属性: N_NAV_TYPE 
        /// </summary>
        [JsonProperty(PropertyName = "c_NAV_TYPE")]
        public string C_NAV_TYPE
        {
            set { c_NAV_TYPE = value; }

            get { return c_NAV_TYPE; }
        }

        /// <summary>
        /// 属性: N_DETAIL 
        /// </summary>
        [JsonProperty(PropertyName = "n_DETAIL")]
        public string N_DETAIL
        {
            set { n_DETAIL = value; }

            get { return n_DETAIL; }
        }

        /// <summary>
        /// 属性: C_KEY_CODE 
        /// </summary>
        [JsonProperty(PropertyName = "c_KEY_CODE")]
        public string C_KEY_CODE
        {
            set { c_KEY_CODE = value; }

            get { return c_KEY_CODE; }
        }

        /// <summary>
        /// 属性: C_KEY_NAME 
        /// </summary>
        [JsonProperty(PropertyName = "c_KEY_NAME")]
        public string C_KEY_NAME
        {
            set { c_KEY_NAME = value; }

            get { return c_KEY_NAME; }
        }

        /// <summary>
        /// 属性: C_IS_SYS 
        /// </summary>
        [JsonProperty(PropertyName = "c_IS_SYS")]
        public string C_IS_SYS
        {
            set { c_IS_SYS = value; }

            get { return c_IS_SYS; }
        }

        /// <summary>
        /// 属性: C_TRU 
        /// </summary>
        [JsonProperty(PropertyName = "c_TRU")]
        public string C_TRU
        {
            set { c_TRU = value; }

            get { return c_TRU; }
        }

        /// <summary>
        /// 属性: C_MODE 
        /// </summary>
        [JsonProperty(PropertyName = "c_MODE")]
        public string C_MODE
        {
            set { c_MODE = value; }

            get { return c_MODE; }
        }

        /// <summary>
        /// 属性: C_RET 
        /// </summary>
        [JsonProperty(PropertyName = "c_RET")]
        public string C_RET
        {
            set { c_RET = value; }

            get { return c_RET; }
        }

    }
}
